tests/features/cascade.feature

```diff
@@ -426,6 +426,13 @@ Feature: Check not HA hosts
           - slot_name: pgconsul_postgresql3_1_pgconsul_pgconsul_net
             slot_type: physical
         """
+        Then <lock_type> "<lock_host>" has following values for key "/pgconsul/postgresql/replics_info"
+        """
+          - client_hostname: pgconsul_postgresql2_1.pgconsul_pgconsul_net
```

Comment:
`postgresql2` is primary, we can't have it as replica

Reply:
nope, pg2 is replica